__tests__/credit-memo.test.ts

```
@@ -0,0 +1,230 @@
+		// Describe the getCreditMemosForDateRange Method
+		it('should fetch credit memos within date range', async () => {
+			// Set the start Date
+			const startDate = new Date('2020-01-09');
+
+			// Set the end Date
+			const endDate = new Date('2022-01-12');
+
+			// Get the List of CreditMemos in that date range for the mock data
+			const creditMemosInDateRange = mockCreditMemoData.filter((creditMemo) => {
+				// Get the CreditMemo Date
+				const creditMemoDate = new Date(creditMemo.MetaData!.LastUpdatedTime!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)